crates/component/src/lib.rs

```diff
@@ -61,19 +62,21 @@ mod test {
             let _ = wasi::config::runtime::get("foo");
             let _ = wasi::config::runtime::get_all();
 
-            wasmcloud::bus::lattice::set_link_name(
+            wasmcloud::bus1_0_0::lattice::set_link_name(
```

Comment:
since this crate is pre-1.0 and this change would be (rust) API breaking anyway, let's only use 2.0.0 version in `wasmcloud_component`?